src/WebJobs.Script/Workers/Http/DefaultHttpWorkerService.cs

```diff
@@ -71,6 +89,29 @@ public Task InvokeAsync(ScriptInvocationContext scriptInvocationContext)
             return ProcessDefaultInvocationRequest(scriptInvocationContext);
         }
 
+        internal async Task ProxyInvocationRequest(ScriptInvocationContext scriptInvocationContext)
+        {
+            try
+            {
+                if (!scriptInvocationContext.TryGetHttpRequest(out HttpRequest httpRequest))
+                {
+                    throw new InvalidOperationException($"Cannot proxy the HttpTrigger function {scriptInvocationContext.FunctionMetadata.Name} without an input of type {nameof(HttpRequest)}.");
+                }
+
+                AddProxyingHeaders(httpRequest, scriptInvocationContext.ExecutionContext.InvocationId.ToString());
+
+                // YARP only requires the destination prefix. The path and query string are added by the YARP proxy during SendAsync using info from the HttpContext.
+                _httpProxyService.StartForwarding(scriptInvocationContext, _destinationPrefix);
+
+                await _httpProxyService.EnsureSuccessfulForwardingAsync(scriptInvocationContext);
+                scriptInvocationContext.ResultSource.SetResult(_successfulInvocationResult);
```

Comment:
Does the downstream code updates this `_successfulInvocationResult` instance? I see we use a shared instance for all invocations.

Reply:
No, the downstream code does not. This value is set to allow the function invocation to continue, but its value is never used (similar to how http proxying flows). We ignore the function invocation result at [this step](https://github.com/Azure/azure-functions-host/blob/2557cf38e1de490f4ea423492ea2ec23fb78f24a/src/WebJobs.Script.WebHost/Middleware/FunctionInvocationMiddleware.cs#L60) so that the actual response just flows through.